bukkit/src/main/java/net/william278/huskhomes/network/RedisMessenger.java

```diff
@@ -4,9 +4,8 @@
 import net.william278.huskhomes.player.BukkitPlayer;
 import net.william278.huskhomes.player.OnlineUser;
 import net.william278.huskhomes.redis.RedisWorker;
+import net.william278.huskhomes.redis.redisdata.RedisPubSub;
 import org.jetbrains.annotations.NotNull;
-import redis.clients.jedis.Jedis;
-import redis.clients.jedis.JedisPubSub;
 
```

Comment:
I'd like to abstract a lot of the networking stuff to the common module for better platform support in the future. I'd like to add an abstract method of having a player send a plugin message and along with that move the PluginMessage and extending RedisMessages classes to common. 

Have a look at what I did with HuskTowns for inspiration: https://github.com/WiIIiam278/HuskTowns/tree/master/common/src/main/java/net/william278/husktowns/network

While arguably a little out of scope for this PR, I'd rather do this work _with_ or alongside this PR in one go to avoid redigging up this code later